apps/www/registry/stories/colors.stories.mdx

```diff
@@ -0,0 +1,94 @@
+import { ColorItem, ColorPalette, Meta } from "@storybook/blocks"
+
+<Meta title="theme/Colors" />
+
+# Colors
+
+The design system color palette consisting of a collection of color scales in a
+range of contrast ratios from the background color.
+
+## Color Palette
+
+Here's the list of variables available for customization:
+
+<ColorPalette>
+  <ColorItem
+    title="background"
+    subtitle="Default background color of <body />...etc"
+    colors={{
+      background: "hsl(var(--background))",
+      oreground: "hsl(var(--foreground))",
```

Comment:
Small mistake

Reply:
Thanks, I updated this in the [color.stories.mdx](https://github.com/shadcn-ui/ui/pull/1561/files#diff-5a2a0349a6b1743fa92ae9708bf85965f06e8c25477413d20290c3197711faf0) for the #1561 👍